src/Donation/DonationRequest.php

```diff
@@ -10,11 +10,23 @@
 use Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumber as AssertPhoneNumber;
 use Ramsey\Uuid\Uuid;
 use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 use Symfony\Component\Validator\Constraints as Assert;
 
+/**
+ * TODO: add endSubscriptionAt in the list when end subscription will be merged.
```

Comment:
Do we have a card for that ?
If yes, it would be better to remove this TODO and add the information in the card IMO.

Reply:
Yes there are an other card to unsubscribe, the idea is to rebase with master when it will be merged^^,

Reply:
Just rebase one of the 2 PR on the other one if they are dependent.

Reply:
I prefer to wait, because I think it's dangerous to be based on an other PR :s

Reply:
some commits will be added and after that it will have a squash, if it's squash , I will have to remove all commits from original rebase and after that rebase on master. And the other thing is I will refactor Donation entity to have all transactions, and add a status with some different values like: confirmation_pending, finished, subscription_in_progress, error.

Reply:
Ok so it's just a matter of organization that you and only you is responsible for. It's not really dangerous. You just need to remember which commits belongs to which branch. Easy to do with a correct naming.

Reply:
Yes, I said dangerous because I could have a miss during manipulation :'( , but yes it's only organization ^^,